src/Schedule.cpp

```diff
@@ -241,6 +241,7 @@ struct FuncScheduleContents {
     MemoryType memory_type = MemoryType::Auto;
     bool memoized = false;
     bool async = false;
+    Expr ring_buffer;
```

Comment:
Worth adding a comment here that this is expected to be a positive int that is the extent of the buffer?